cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionCheckCommand.java

```diff
@@ -266,6 +267,12 @@ public class ExtractionCheckCommand extends Command {
       description = "Full git commit sha, used for setting a status on a commit in Github.")
   String commitSha;
 
+  @Parameter(
+      names = {"--exclude-files-regex", "-efr"},
+      arity = 1,
+      description = "Regex to exclude files from context comment check")
+  String excludedFilesRegex;
```

Comment:
As this is limited to the context comment checker I would put the configuration in checker options rather than the top level command.

Reply:
Done